src/routes/chart/update-panels.ts

```diff
@@ -22,10 +22,10 @@ type Panel = {
 };
 
 type Request = {
-  body: Record<KindsResources, Array<{
+  body: Partial<Record<KindsResources, Array<{
```

Comment:
I'm not sure if this is what we want; how can the api update panels if the contents of the request body is optional?

Reply:
here it will update nothing, I think